src/modules/85-cv/components/ChangeTimeline/__tests__/ChangeTimeline.test.tsx

```diff
@@ -30,6 +30,23 @@ jest.mock('services/cv', () => ({
       loading: false,
       cancel: jest.fn()
     }
+  }),
+  useGetMonitoredServiceChangeTimeline: jest.fn().mockImplementation(() => {
+    return {
+      data: {
+        resource: {
+          categoryTimeline: {
+            Deployment: [],
+            Infrastructure: [],
+            Alert: []
+          }
+        }
+      },
+      refetch: jest.fn(),
+      error: null,
+      loading: false,
+      cancel: jest.fn()
+    }
   })
```

Comment:
You can simplify this a wee bit
```suggestion
  useGetMonitoredServiceChangeTimeline: jest.fn().mockReturnValue({
    data: {
      resource: {
        categoryTimeline: {
          Deployment: [],
          Infrastructure: [],
          Alert: []
        }
      }
    },
    refetch: jest.fn(),
    error: null,
    loading: false,
    cancel: jest.fn()
  })
```